website/docs/engine-flink/delta-joins.md

```diff
@@ -145,7 +145,7 @@ The delta join feature is introduced since Flink 2.1 and still evolving, and its
 Refer to the [Delta Join Issue](https://issues.apache.org/jira/browse/FLINK-37836) for the most up-to-date information.
 
 
-### Flink 2.1
+### Flink 2.1.2 and later
```

Comment:
I suggest keeping “Flink 2.1” as the title, but adding a note beneath it, otherwise, it's truly strange we are only linking to a unreleased version, the note can also help users better understand the issue:

```
:::warning
There is a known issue ([FLINK-XXXX ](link)) in Flink ≤ 2.1.1 that prevents certain queries from being translated into delta joins. This has been fixed in Flink 2.1.2 and Flink 2.2.
:::
```